lib/ancestry/instance_methods.rb

```diff
@@ -263,6 +263,24 @@ def descendant_ids depth_options = {}
     end
 
     def descendant_of?(node)
+      ancestor_ids[0..-2].include?(node.id)
```

Comment:
This looks like the definition of `indirect_of`. no?

Reply:
oh shoot - I fixed that